cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -551,6 +551,16 @@ protected VolumeMount createVolumeMount(String name, String path) {
         return volumeMount;
     }
 
+    protected VolumeMount createVolumeMountWithSubPath(String name, String path, String subPath) {
```

Comment:
Do we really need two methods here? The old one without subPath and the new one with subPath? What does it mean when subPath is null? It calling `.withSubPath(null)` doesn't change anything, we can probably just use single method, or?

Reply:
The two parameter method is used in many places.  I've updated the implementation of the two parameter method to invoke the three parameter method with a null subPath.